tests/basics/namedtuple1.py

```diff
@@ -82,3 +82,12 @@
 # Not implemented so far
 #T2 = namedtuple("TupComma", "foo,bar")
 #t = T2(1, 2)
+
+try:
+    d = T(5, 6)._asdict()
```

Comment:
There should be explicit check for _asdict attribute present, leading to SKIP otherwise.

Reply:
You mean a seperate namedtuple2.py with 'standard' print('SKIP') and raise SystemExit if _asdict isn't present?

Reply:
I actually meant how skip check should be done, but good catch, it should be a separate test.